packages/overlayscrollbars-react/package.json

```
@@ -39,8 +39,8 @@
         "typescript": "^3.8.3"
     },
     "peerDependencies": {
-        "react": "^16.4.0 || ^17.0.0",
-        "overlayscrollbars": "^1.10.0"
+        "overlayscrollbars": "^1.10.0",
+        "react": "^18.0.0"
```

```suggestion
        "react": "^16.4.0 || ^17.0.0 || ^18.0.0"
```

To keep the compatibility to older react versions